Exercise_9/src/MyFramework/customTags/CustomLink.js

```diff
@@ -0,0 +1,19 @@
+const customlink = {
+  template: '<a></a>',
+  beforeMount: (app, element, componentData) => {
+    for (let index = 0; index < componentData.length; index += 1) {
+      const attributeName = componentData[index].name;
+      const attributeValue = componentData[index].value;
+
+      if (attributeName === 'href') {
+        element.addEventListener('click', () => {
+          app.navigate(attributeValue.substring(1));
+        }, false);
+      } else {
+        element.setAttribute(attributeName, attributeValue);
```

Comment:
As I proposed in the comment above, it is better to have a common implementation for this method. In your case, the only custom logic is to navigate on click.